__tests__/app.test.js

```diff
@@ -247,5 +238,56 @@ describe("GET /api/reviews/:review_id with comment_count", () => {
           })
         );
       });
-  })
-})
\ No newline at end of file
+  });
+});
+
+// GET /API/REVIEWS
+
+describe("GET /api/reviews", () => {
+  test("status: 200, responds with an array of review objects", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        expect(reviews).toBeInstanceOf(Array);
+        expect(reviews).toHaveLength(13);
+
+        reviews.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: expect.any(Number),
+              category: expect.any(String),
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              review_body: expect.any(String),
```

Comment:
We aren't going to expect the revie_body back

Reply:
It was mentioned on the ticket so I thought it needed to be added there, I'll remove it!